src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:neo4j:bolt://localhost
+spring.datasource.driver-class-name=org.neo4j.jdbc.bolt.BoltDriver
+spring.datasource.username=neo4j
+spring.datasource.password=toto
```

Comment:
You can override it when you start the application via the ways described in https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html ;-)